src/Sylius/Bundle/CoreBundle/Doctrine/ORM/CartRepository.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Doctrine\ORM;
+
+use Sylius\Bundle\CartBundle\Doctrine\ORM\CartRepository as BaseCartRepository;
+use Sylius\Component\Channel\Model\ChannelInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Repository\CartRepositoryInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class CartRepository extends BaseCartRepository implements CartRepositoryInterface
```

Comment:
I hate this `Order` -> `Cart` -> `Order` design, but as far as I'm familiar with this concept in `Core` we should have `Order`, so this should be renamed to `OrderRepository`... 😕